src/zoid/buttons/util.js

```diff
@@ -222,3 +223,60 @@ export function applePaySession() : ?ApplePaySessionConfigRequest {
         return undefined;
     }
 }
+
+export function getButtonSize(props : ButtonProps, container : string | HTMLElement | void) : string | void {
+    if (!container) {
+        return;
+    }
+
+    let containerWidth = 0;
+
+    if (container && typeof container !== 'string') {
```

Comment:
Are we able to remove these `container` truthy checks since we added the falsy check above?
```suggestion
    if (typeof container !== 'string') {
```